WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -70,19 +58,7 @@ public JsonRpcResultList GetCoinList()
 		{
 			throw new ArgumentException($"{nameof(activeWallet.Coins)} was not {typeof(CoinsRegistry)}.");
 		}
-		return coinRegistry.AsAllCoinsView().Select(
-			x => new JsonRpcResult
-			{
-				["txid"] = x.TransactionId.ToString(),
-				["index"] = x.Index,
-				["amount"] = x.Amount.Satoshi,
-				["anonymityScore"] = x.HdPubKey.AnonymitySet,
-				["confirmed"] = x.Confirmed,
-				["confirmations"] = x.Confirmed ? serverTipHeight - (uint)x.Height.Value + 1 : 0,
-				["keyPath"] = x.HdPubKey.FullKeyPath.ToString(),
-				["address"] = x.HdPubKey.GetAddress(Global.Network).ToString(),
-				["spentBy"] = x.SpenderTransaction?.GetHash().ToString()
-			}).ToImmutableArray();
+		return coinRegistry.AsAllCoinsView().Select(x => GetRpcResultFromSmartCoin(x, serverTipHeight)).ToImmutableArray();
```

Comment:
I suggest to use Map prefix instead of Get for mapping entity methods.
MapSmartCoinToJsonRpcResult or MapSmartCoinToResult